pallets/motion/src/benchmarking.rs

```diff
@@ -19,36 +19,42 @@ mod benchmarks {
 	use super::*;
 
 	#[benchmark]
-	fn simple_majority() {
+	fn simple_majority() -> Result<(), BenchmarkError> {
 		let call: <T as Config>::RuntimeCall = frame_system::Call::remark { remark: vec![] }.into();
-		let origin = <T as Config>::SimpleMajorityOrigin::try_successful_origin().unwrap();
+		let origin = <T as Config>::SimpleMajorityOrigin::try_successful_origin()
```

Comment:
As per the configuration in our runtime, this check will never ensure origin. Hence, the benchamrk needed to be updated such that a possible runtime config won't break it.

I have configure this case to use `Weightless`. Maybe MAX weight could be a better option.

Same with the rest of changes below.

Reply:
How do you then create a weight value for this ?